packages/bpk-component-calendar/src/BpkCalendarContainer.tsx

```diff
@@ -66,6 +66,7 @@ export type Props = {
   initiallyFocusedDate?: Date | null;
   markToday?: boolean;
   markOutsideDays?: boolean;
+  monthHeight?: number;
```

Comment:
We don't need to specify this prop specifically as the `<Calendar>` component passes the rest of the props via `{...calendarProps}`

See [here](https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-calendar/src/BpkCalendarContainer.tsx#L391)